get-resource.sh

```diff
@@ -36,7 +36,9 @@ if [ -s "/shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENA
     echo "$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum found, contents:"
     cat /shared/html/images/$RHCOS_IMAGE_FILENAME_OPENSTACK/$RHCOS_IMAGE_FILENAME_COMPRESSED.md5sum
 else
-    curl -g --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}"
+    for max_tries in {1..5}; do
+        curl -g --insecure --compressed -L -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}" && break || echo "Download failed, retrying..."
+    done
```

Comment:
maybe add a sleep between runs?

Reply:
Yes, but for how many seconds? Consider that curl timeout takes a while anyway

Reply:
Done

Reply:
Every time I see eval I get nervous - what about `for i in $(seq 5)`

Also 30s is quite a long sleep time, why not sleep for ` $(($i*$i))` as a crude exponential backoff, then if it's just one retry we'll only wait a second?

It also would be easier to read if you used an if statement instead of &&/|| IMO

Reply:
The idea was to make the sleeping time configurable, but also your suggestions seems fine.